umbraco/Umbraco-CMS
# V13: website assets directly from static assets

### Description

- Configured pipeline to check out submodules
  - Thereby the `npm ci` for new backoffice works, and fixes an issue where this was skipped on pipelines
 - Moved static assets required for website rendering of Umbraco pages
   - No content
   - 404
   - Maintenance page
   
### Test
   - Do a clean on solution
     - Verify the `src/Umbraco.CMS.StaticAssets/wwwroot/umbraco/website`-folder still exists
   - Do a build on solution
     - Verify the new backoffice now exists
        - `src/Umbraco.CMS.StaticAssets/wwwroot/umbraco/backoffice`
     - Verify the old backoffice exists 
        - `src/Umbraco.CMS.StaticAssets/wwwroot/umbraco/*` 
          - Except `/website` and `/backoffice`
      - Verify the you can run umbraco from `Umbraco.Web.UI`
      - Verify the you can run umbraco from `Umbraco.Web.UI.New`
   - Do a build
     - Verify the static assets are not built again (Should be a lot faster this time)
